web/src/main/webapp/WEB-INF/jsp/team/match/goal/create.jsp

```diff
@@ -0,0 +1,93 @@
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="true" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib prefix="sec" uri="http://www.springframework.org/security/tags" %>
+
+<my:pagetemplate title="UpcommingMatchesOverview">
+    <jsp:attribute name="body">
+
+        <div class="container">
+            <div class="row">
+
+                <div class="col-md-8 offset-md-2">
+                    <form:form class="form-horizontal match-form" id="goal-form" method="post" modelAttribute="goalRequest" action="${requestScope['javax.servlet.forward.request_uri']}" autocomplete="off">
+                        <fieldset>
+                            <legend id="match-form-legend">New goal creation form</legend>
+
+                            <s:bind path="scorerId">
+                                <div class="form-group row match-form-item ${status.error ? 'has-danger' : ''}">
+                                    <form:label path="scorerId" for="scorer" cssClass="col-md-3 col-form-label col-form-label-lg form-control-label">Scorer</form:label>
+                                    <div class="col-md-7">
+                                         <form:select path="scorerId" id="scorer" cssClass="match-dropdown-items match-form-team form-control">
+                                             <c:choose>
+                                                 <c:when test="${goalRequest.scorerId != null}">
+                                                     <form:option value="${null}" label="--- Select ---" disabled="true"/>
+                                                     <c:forEach items="${players}" var="scorer">
+                                                         <form:option value="${scorer.id}" label="${scorer.firstname} ${scorer.surname}" selected="${scorer.id.equals(goalRequest.scorerId) ? 'selected' : ''}"/>
```

Comment:
na zaklade vstupnich parametru neni jak otestovat, zda je dany strelec ci asistent golu ze spravneho tymu, musel by se pridat atribut "skorujici tym", pak by to bylo proveditelne, tedy to udelam...